mlrun/datastore/v3io.py

```diff
@@ -91,8 +95,34 @@ def get_storage_options(self):
             v3io_api=mlrun.mlconf.v3io_api,
         )
 
+    def upload_helper(self, key: str, src_path: str, max_chunk_size: int = ONE_GB):
+        """helper function for upload method, allows for controlling max_chunk_size in testing"""
```

Comment:
You can just make it private by calling it `_upload`. Then you don't have to document it either (devs understand it and users aren't exposed to it), unless you really want to. :) The test will still be able to access it.

Reply:
changed